### DISCUSSION TITLE
User config for SetOption73

### DISCUSSION THREAD

**Discussion by cociweb** [arendst/Tasmota#11194] [Q&A] [answered]
Hello,

I'm relativelly new with Tasmota, but it's a very handy firmware.  First of all let me congratulate for the developers!

I would like to build and customize  my own devices. I'm using the [SetOption73](https://tasmota.github.io/docs/Commands/#setoptions) instead of direct control of the relay with the interwork of the multi-press feature.

I've investigated and read for hours about this parameter (SetOption73) but without any luck.
I would like to pass this parameter into the firmware with the help of the `user_config_override.h` file

Do you have any suggestion how to do it? Is it implemented or should I open a feature request?
(Note: the pair of the SO73 is the SO114 for switch and it's missed as well.)

Thank's in advance.
Br,
cociweb

**Comment by sfromis** [comment, marked as answer]
SetOption73 and SetOption114 are not among the flags currently having a compile-time #define for modification in `user_config_override.h`

Flag settings implementing default #define for other SetOption flags are initialized in `SettingsDefaultSet2` of `settings.ino`

**Comment by nagyrobi** [comment]
Also needed for SetOption89: https://github.com/arendst/Tasmota/discussions/11195

**Comment by barbudor** [comment]
I'm on it
Any other one before I PR ?

**Reply by nagyrobi** [reply]
Thanks, man!

**Reply by barbudor** [reply]
https://github.com/arendst/Tasmota/pull/11204

**Reply by cociweb** [reply]
What a great and quick implementation! 

Thank you guys!
